spec/requests/tenant_quotas_spec.rb

```diff
@@ -131,6 +131,12 @@
       expect(response).to have_http_status(:no_content)
     end
 
+    it "supports OPTIONS requests on a subcollection without authorization" do
+      options "/api/tenants/#{tenant.id}/quotas"
```

Comment:
use the rails helper for building the url, api_tenant_quotas_url

Reply:
@abellotti OK will do. Thank you!